RubberduckTests/Grammar/AnnotationTests.cs

```diff
@@ -39,7 +39,7 @@ public void ModuleCleanupAnnotation_TypeIsModuleCleanup()
         [Test]
         public void TestMethodAnnotation_TypeIsTestTest()
         {
-            var annotation = new TestMethodAnnotation(new QualifiedSelection(), null);
+            var annotation = new TestMethodAnnotation(new QualifiedSelection(), new[] { "param" });
```

Comment:
I don't think this change should happen - I think that might have been the result of using `First` rather than `FirstOrDefault`.